pkg/sentry/fs/proc/net.go

```diff
@@ -310,6 +311,79 @@ func (n *netSnmp) ReadSeqFileData(ctx context.Context, h seqfile.SeqHandle) ([]s
 	return data, 0
 }
 
+// netRoute implements seqfile.SeqSource for /proc/net/route.
+//
+// +stateify savable
+type netRoute struct {
+	s inet.Stack
+}
+
+// NeedsUpdate implements seqfile.SeqSource.NeedsUpdate.
+func (n *netRoute) NeedsUpdate(generation int64) bool {
+	return true
+}
+
+// ReadSeqFileData implements seqfile.SeqSource.ReadSeqFileData.
+// See Linux's net/ipv4/fib_trie.c:fib_route_seq_show.
+func (n *netRoute) ReadSeqFileData(ctx context.Context, h seqfile.SeqHandle) ([]seqfile.SeqData, int64) {
+	if h != nil {
+		return nil, 0
+	}
+
+	interfaces := n.s.Interfaces()
+	contents := []string{"Iface\tDestination\tGateway\tFlags\tRefCnt\tUse\tMetric\tMask\tMTU\tWindow\tIRTT"}
+	for _, rt := range n.s.RouteTable() {
+		if rt.Family != linux.AF_INET {
+			continue
+		}
+
+		if rt.Type != linux.RTN_UNICAST {
+			continue
+		}
```

Comment:
I believe a comment here describing why AF_INET/RTN_UNICAST routes are being skipped would be useful.

Reply:
This kind of check comes from "Linux's net/ipv4/fib_trie.c:fib_route_seq_show" which we have referenced before this function. May inline comment can be not necessary?

BTW, just rechecked the Linux code, seems that the checks shall be below instead:
```
if ((fa->fa_type == RTN_BROADCAST) ||
    (fa->fa_type == RTN_MULTICAST))
         continue;
```

Reply:
// /proc/net/route does not include broadcast or multicast routes.